Knoema.Client/Data/TimeSeriesRawData.cs

```diff
@@ -19,10 +19,12 @@ internal void OnDeserialized(StreamingContext context)
 		{
 			Dimensions = _dimensions.Select(pair =>
```

Comment:
since we've got different sorts of stuff in _dimensions it shall be renamed to reflect it's purpose

Reply:
done